sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/InMemoryCatalog.scala

```diff
@@ -73,6 +79,8 @@ class InMemoryCatalog extends ExternalCatalog {
     }
   }
 
+  private val fs = FileSystem.get(new Configuration)
```

Comment:
I think it is better to use `sparkContext.hadoopConfiguration` (we can access sparkContext in SharedState). We can use `new Configuration` as a default value of the hadoop conf to avoid of changing those tests.